### DISCUSSION TITLE
Isn't there any field for s3 remote logging in values.yaml of helm chart?

### DISCUSSION THREAD

**Discussion by sgc109** [apache/airflow#21453] [Q&A]
Hi, I'm using KubernetesExecutor with official helm chart. 

I'd like to use s3 as a remote logging storage, but I cannot find any values.yaml field in official helm chart, but [only for elasticsearch](https://github.com/apache/airflow/blob/0a3ff43d41d33d05fb3996e61785919effa9a2fa/chart/values.yaml#L1386), even though there is a [configuration field](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html?highlight=remote%20log#remote-base-log-folder) for s3 remote logging. Could I get any guides for this?

**Comment by potiuk** [comment]
You should configure it manually based on https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/logging/s3-task-handler.html
